src/main/java/org/opentripplanner/netex/NetexBundle.java

```diff
@@ -77,7 +81,9 @@ public OtpTransitServiceBuilder loadBundle(
 
         // init parser and mapper
         xmlParser = new NetexXmlParser();
-        mapper = new NetexMapper(transitBuilder, netexFeedId, deduplicator, issueStore);
+        mapper = new NetexMapper(transitBuilder, netexFeedId, deduplicator, issueStore,
+                ferryIdsNotAllowedForBicycle
+        );
```

Comment:
We try to keep stuff with the same abstraction level at the same indentation level - so it is easy to see where something begin and ends. So the above code can be formatted:
```java
    mapper = new NetexMapper(
            transitBuilder, netexFeedId, deduplicator, issueStore, ferryIdsNotAllowedForBicycle
    );
```
or
```java
    mapper = new NetexMapper(
            transitBuilder,
            netexFeedId,
            deduplicator,
            issueStore,
            ferryIdsNotAllowedForBicycle
    );
```